### ISSUE TITLE
Doc parse nit during rails 3.2.13.rc1 install

### ISSUE THREAD

**Issue by fburleigh** [jimweirich/builder#39]
At the OS X terminal, I see this:
- Parsing documentation for builder-3.0.4
  ruby-2.0.0-p0/lib/ruby/2.0.0/rdoc/parser.rb:87: warning: Unsupported encoding : ignored
  ruby-2.0.0-p0/lib/ruby/2.0.0/rdoc/parser.rb:87: warning: Unsupported encoding ignored

Additionally, the Builder installed with that rails is 3.0.2.

(Edit to fix copy/paste error)

**Comment by deivid-rodriguez** [comment]
Same issue here using last rdoc and last builder

```
davidr@pantani:~/Proyectos$ gem install builder
Fetching: builder-3.2.0.gem (100%)
Successfully installed builder-3.2.0
Parsing documentation for builder-3.2.0
/home/davidr/.rvm/gems/ruby-2.0.0-p0/gems/rdoc-4.0.1/lib/rdoc/parser.rb:87: warning: Unsupported encoding : ignored
/home/davidr/.rvm/gems/ruby-2.0.0-p0/gems/rdoc-4.0.1/lib/rdoc/parser.rb:87: warning: Unsupported encoding  ignored
/home/davidr/.rvm/gems/ruby-2.0.0-p0/gems/rdoc-4.0.1/lib/rdoc/parser.rb:87: warning: Unsupported encoding : ignored
/home/davidr/.rvm/gems/ruby-2.0.0-p0/gems/rdoc-4.0.1/lib/rdoc/parser.rb:87: warning: Unsupported encoding  ignored
Installing ri documentation for builder-3.2.0
Done installing documentation for builder after 0 seconds
1 gem installed
```

**Comment by jimweirich** [comment]
I comment in XmlBase.rb was causing RDoc grief.  I changed the comment so that RDoc no long complains.  FIxed in head and pushed in version 3.2.2.
